recipe/meta.yaml

```diff
@@ -11,9 +11,11 @@ source:
     - 0001-support-poppler-0.64.0.patch
 
 build:
-  number: 8
+  number: 9
   skip: True  # [win]
   detect_binary_files_with_prefix: true
+  ignore_run_exports_from:
+    - perl 5.*
```

Comment:
@conda-forge/texlive-core, I did not check how `perl` is used in this build.
If possible, I'd suggest to move it from `requirements/host` to `requirements/build`. We can then also omit this `ignore_run_exports_from`.

Reply:
AFAICT the build just checks whether there is a `perl` binary available to decide if it should or shouldn't copy some Perl scripts (but not actual Perl packages). So having it in `requirements/host` with the `ignore_run_exports_from` should be fine.